eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -25,6 +25,18 @@ steps:
 
       displayName: Copy ${{ parameters.DirectoryToSync }} from azure-sdk-tools to ${{ repo }}
       workingDirectory: $(System.DefaultWorkingDirectory)
+    
+    - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
This seems likes it belongs in the wrapper yml file not the general sync-directory as this might be used by other repos as well.

Reply:
This should go in https://github.com/Azure/azure-sdk-tools/blob/master/eng/pipelines/eng-common-sync.yml.